src/components/admin/MainIssueController.jsx

```diff
@@ -215,7 +216,7 @@ export default class MainIssueController extends FalcorController {
             window.alert(`${article.title} has no authors. Please correct this`);
             return false;
           }
-          if (/http(?!s)/.test(article.html)) {
+          if (hasNonHttpsURL(article.html)) { // if the function returns true, the url is correct
```

Comment:
1. I don't think we need the comment anymore as the function name is super clear (yay!)

2. I think the comment is incorrect :P, unless by correct you mean "has a problem we need to warn the editors about"